### DISCUSSION TITLE
Remove /give.

### DISCUSSION THREAD

**Discussion by IoIxD** [EssentialsX/Essentials#5171] [Ideas and Feature Suggestions] [answered]
> EssentialsX provides enhanced alternatives to several default commands. For example, EssentialsX's `/give` supports a user-friendly item meta syntax that's less awkward to use than vanilla NBT...

NBT, inherently, is not user friendly. In minecraft's /give command it's literally JSON, a format that wasn't really intended to be written out by humans anyways, let alone one that was meant to be typed in one line (that said, JSON is at least a format that can be written in one line. The other formats for config files don't really make sense to use in a command). 

Unless there's something I'm missing, your idea of "making the command less awkward"...is just swapping around the item meta argument and the item amount argument. That doesn't fix anything. The command is still awkward to type* but now I've spent the past several months appending `minecraft:` to the give command every time, because it didn't even occur to me how else you could do the command and I thought this command was a leftover from before Minecraft added it's own `/give` command in 1.3 that nobody else has thought to complain about.

If there is nothing I'm missing, I ask (on behalf of not only me but my admins as well) to just remove it, and I would like to ask if anyone reading this other then the developers has actually gotten used to this. And if there is something I'm missing then honestly I would ask for a configuration option to remove the /give replacement instead of having to put `minecraft:` in front of give every time, because Minecraft's `/give` command works just fine for me and I can't really imagine myself getting used to new syntax for whatever Essentials adds.

<sup>*and that assumes you don't just use a generator or copy paste for your command, which exist for many situations and I find myself using frequently</sup>

**Comment by oskarkk** [comment]
You can disable any Essentials command here: https://github.com/EssentialsX/Essentials/blob/2.x/Essentials/src/main/resources/config.yml#L173

If you disable it there, `/give` will just default to the normal vanilla minecraft version. The other way is setting it in `aliases.yml` but I think that would break command completion.

**Comment by IoIxD** [comment]
That's good to know. 

I still think this is kind of silly and would love to talk to somebody who prefers this.

**Comment by mdcfe** [comment, marked as answer]
`/give` provides shorthand aliases for almost all items in the game (see `/itemdb` for aliases of other items), as well as a human-friendly [Item Meta](http://ess.khhq.net/wiki/Item_Meta) syntax that is easier to write than NBT. This system is also used by `/item`, kits and item trades to provide consistent item syntax across the plugin, and is familiar to many long-term EssentialsX users.

If you want, you can disable individual commands as mentioned by oskarkk and restore the vanilla `/give` command. This option is provided so that new users can choose which command they want to use.
